pkg/oauth/apis/oauth/types.go

```diff
@@ -4,6 +4,9 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// +genclient
+// +genclient:nonNamespaced
```

Comment:
i was wrong, oauth use generic storage as you noted and we got all verbs for free, so this is correct and the legacy client was left behind.